climada/hazard/tc_tracks.py

```diff
@@ -2980,3 +2980,176 @@ def _zlib_from_dataarray(data_var: xr.DataArray) -> bool:
     if np.issubdtype(data_var.dtype, float) or np.issubdtype(data_var.dtype, int):
         return True
     return False
+
+
+def compute_track_density(
+    tc_track: TCTracks,
+    res: int = 5,
+    genesis: bool = False,
+    norm: str = None,
+    filter_tracks: bool = True,
+    wind_min: float = None,
+    wind_max: float = None,
+) -> tuple[np.ndarray, tuple]:
+    """Compute absolute and normalized tropical cyclone track density. Before using this function,
+    apply the same temporal resolution to all tracks by calling :py:meth:`equal_timestep` on the
+    TCTrack object. Due to the computational cost of the this function, it is not recommended to
+    use a grid resolution higher tha 0.1°. First, this function creates 2D bins of the specified
+    resolution (e.g. 1° x 1°). Second, since tracks are not lines but a series of points, it counts
+    the number of points per bin. Lastly, it returns the absolute or normalized count per bin.
+    To plot the output of this function use :py:meth:`plot_track_density`.
+
+    Parameters:
+    ----------
+    tc_track: TCT track object
+        track object containing a list of all tracks
+    res: int (optional), default: 5°
+        resolution in degrees of the grid bins in which the density will be computed
+    genesis: bool, Default = False
+        If true the function computes the track density of only the genesis location of tracks
+    norm: bool (optional), default = False
+        If False it returns the number of samples in each bin. If True, returns the
+        specified normalization function at each bin computed as count_bin / grid_area.
```

Comment:
This docstring should be updated, since the `norm` parameter now is a string.

Reply:
Also, you might want to specify the possible values that `norm` can take somewhere in the code (e.g., as a constant) and to check in this function whether the value passed by the user is allowed.

Reply:
Yes thanks!